packages/front-end/components/Features/RuleModal.tsx

```diff
@@ -69,6 +74,26 @@ export default function RuleModal({
       header={rules[i] ? "Edit Override Rule" : "New Override Rule"}
       submit={form.handleSubmit(async (values) => {
         const ruleAction = i === rules.length ? "add" : "edit";
+
+        if (values.scheduleRules.length) {
```

Comment:
I think you can remove the length check and just do:
`values.scheduleRules?.foreach(`